charts/rancher-turtles/values.yaml

```diff
@@ -8,14 +8,14 @@ rancherTurtles:
   features:
     cluster-api-operator:
       cleanup: true
-      kubectlImage: registry.k8s.io/kubernetes/kubectl:v1.28.0
+      kubectlImage: registry.k8s.io/kubernetes/kubectl:v1.30.0
     embedded-capi:
       disabled: true
     rancher-webhook:
       cleanup: true
-      kubectlImage: registry.k8s.io/kubernetes/kubectl:v1.28.0
+      kubectlImage: registry.k8s.io/kubernetes/kubectl:v1.30.0
     rancher-kubeconfigs:
-      label: true
```

Comment:
We can remove this option entirely, to avoid situation when the kubeconfig might not get labeled.